src/app.js

```diff
@@ -1,16 +1,29 @@
-const express = require("express");
+const LogStream = require("./myStream");
 const bodyParser = require("body-parser");
+const errorHandler = require("./error-handler");
+const express = require("express");
+const morgan = require("morgan");
 const repositoryCotroller = require("./controllers/repository");
+const userCotroller = require("./controllers/user");
+const { HttpNotFound } = require("./utils/errors");
```

Comment:
```suggestion
const { HttpNotFound } = require("./errors");
```

Geralmente a ideia de "utils" eh bem ruim IMO. Eh o tipo de coisa que vai se tornar o buraco negro do projeto, tudo acaba caindo ali dentro. E no geral sempre tem um conceito mais especifico que se aplica as coisas.

Nesse caso aqui, uma pasta "errors" na raiz já seria o suficiente.